editor.planx.uk/src/pages/FlowEditor/components/PreviewBrowser.tsx

```diff
@@ -196,19 +204,39 @@ function AlteredNodesSummaryContent(props: any) {
       {changeSummary["portals"].length > 0 && (
         <Box pt={2}>
           <Typography variant="body2">{`This includes recently published changes in the following nested services:`}</Typography>
-          <ul>
+          <List sx={{ listStyleType: "disc", marginLeft: 4 }}>
             {changeSummary["portals"].map((portal, i) => (
-              <li key={i}>
-                {useStore.getState().canUserEditTeam(portal.split("/")[0]) ? (
-                  <Link href={`../${portal}`} target="_blank">
-                    <Typography variant="body2">{portal}</Typography>
-                  </Link>
-                ) : (
-                  <Typography variant="body2">{portal}</Typography>
-                )}
-              </li>
+              <ListItem key={i} disablePadding sx={{ display: "list-item" }}>
+                <ListItemText
+                  primary={
+                    useStore
+                      .getState()
+                      .canUserEditTeam(portal.text.split("/")[0]) ? (
+                      <Link href={`../${portal.text}`} target="_blank">
+                        <Typography variant="body2">{portal.text}</Typography>
+                      </Link>
+                    ) : (
+                      <Typography variant="body2">{portal.text}</Typography>
+                    )
+                  }
+                  secondary={
+                    <>
+                      <Typography variant="body2" fontSize="small">
+                        {`Last published ${formatDistanceToNow(
+                          new Date(portal.publishedAt),
+                        )} ago by ${portal.publishedBy}`}
```

Comment:
This is currently a userId, and not firstname + lastname. As discussed, this is because the info is generated from a public query.

I think having just a number is probably more confusing for Editors? I'd say we may be best not including this just now as MVP, and if requested we can think of how we can get this info in a secure way?

Reply:
Correct - `dataMerged` makes a public query to the `flows` & `published_flows` table which is where the flattened metadata is sourced from.

Once we're reading it in the Editor `PreviewBrowser` here, we will be able to make an authenticated call to lookup user details though ! It's on my final-tidy to-do list before opening for testing once I'm back to one branch :white_check_mark: